### ISSUE TITLE
Dependency issues when installing via npm

### ISSUE THREAD

**Issue by vitosamson** [bustlelabs/mobiledoc-kit#278]
The commonjs build `require`s some dependencies that are only listed as dev dependencies in package.json and don't get installed with `npm install mobiledoc-kit` -

```
ERROR in ./~/mobiledoc-kit/dist/commonjs/mobiledoc-kit/editor/editor.js
Module not found: Error: Cannot resolve module 'content-kit-utils' in /Users/samson/Desktop/dev/wysiwyg/react-content-kit-editor/node_modules/mobiledoc-kit/dist/commonjs/mobiledoc-kit/editor
 @ ./~/mobiledoc-kit/dist/commonjs/mobiledoc-kit/editor/editor.js 29:23-51

ERROR in ./~/mobiledoc-kit/dist/commonjs/mobiledoc-kit/utils/paste-utils.js
Module not found: Error: Cannot resolve module 'mobiledoc-html-renderer' in /Users/samson/Desktop/dev/wysiwyg/react-content-kit-editor/node_modules/mobiledoc-kit/dist/commonjs/mobiledoc-kit/utils
 @ ./~/mobiledoc-kit/dist/commonjs/mobiledoc-kit/utils/paste-utils.js 13:29-63

ERROR in ./~/mobiledoc-kit/dist/commonjs/mobiledoc-kit/utils/paste-utils.js
Module not found: Error: Cannot resolve module 'mobiledoc-text-renderer' in /Users/samson/Desktop/dev/wysiwyg/react-content-kit-editor/node_modules/mobiledoc-kit/dist/commonjs/mobiledoc-kit/utils
 @ ./~/mobiledoc-kit/dist/commonjs/mobiledoc-kit/utils/paste-utils.js 15:29-63
```

**Comment by mixonic** [comment]
@vitosamson Are you getting good builds if those deps are made normal "dependencies"?

Is this for webpack or node?

**Comment by mixonic** [comment]
@vitosamson I'll be near a computer until Tuesday next week- Would like to get this settled for you if possible.

**Comment by vitosamson** [comment]
I just manually `npm install`ed the missing deps. Adding them to `dependencies` should resolve the issue.

**Comment by vitosamson** [comment]
This was for webpack but the issue should present itself in either environment as the dependencies just weren't getting installed.

**Comment by mixonic** [comment]
The build options in https://github.com/bustlelabs/mobiledoc-kit/blob/master/Brocfile.js#L16-L20 apply to all builds- so I would expect that the dependencies are compiled in as they are for the globals build (for instance).

/cc @bantic

**Comment by vitosamson** [comment]
The problem seems to be that the commonjs build requires those dependencies. For example, dist/commonjs/mobiledoc-kit/editor/editor.js contains `var _contentKitUtils = require('content-kit-utils');`, dist/commonjs/mobiledoc-kit/utils/paste-utils contains `var _mobiledocHtmlRenderer = require('mobiledoc-html-renderer');` and `var _mobiledocTextRenderer = require('mobiledoc-text-renderer');`

**Comment by bantic** [comment]
@vitosamson I've released version 0.8.0-beta.1 of mobiledoc-kit. It should fix those dependency issues for you. Please reopen this if not!

**Comment by vitosamson** [comment]
Looks good now, thanks!
